docs/report/report.tex

```diff
@@ -731,24 +731,75 @@ \subsubsection*{Meilenstein 4} $~$ \\
 Um insgesamt auch die Arbeit kontinuierlicher zu gestalten, sollten Aufgaben in kleinere Probleme aufgeteilt werden.
 
 \subsubsection*{Meilenstein 5} $~$ \\
-Der Meilenstein fünf vom 14.01.2021 bis zum 28.01.2021 sollte die Entwicklung am Produkt weitgehend zu abschließen, sodass sich das Team auf den Bericht fokussieren konnte.
+Mit dem fünften Meilenstein sollte zwischen dem 14.01.2021 und 28.01.2021 die Entwicklung am Produkt weitgehend abgeschlossen werden, sodass der Fokus mehr auf den Bericht gerichtet werden konnte.
+
 %Ziel
-Geplant war, die letzten Anwendungszenarien final umzusetzen, sodass die letzten beiden Meilensteine höchstens zum Beheben kleinerer Fehler benötigt werden würden.
+Geplant war hierbei, die letzten Anwendungszenarien final umzusetzen, sodass die letzten beiden Meilensteine höchstens zum Beheben kleinerer Fehler benötigt genutzt werden müsste.
 Somit war das Ziel die Umsetzung einer Oberfläche für die Einstellungen der \textit{Leek Box} selbst als auch der einzelnen Tags.
 Die User Interface Elemente sollten, sofern noch nicht geschehen, ein einheitliches Design erhalten.
 Auch sollte die Dokumentation zur Bedienung der \textit{Leek Box} fertiggestellt sein und die Arbeit für den Bericht begonnen werden.
+
+% Durchführung
+Die Einstellungen der \textit{Leek Box} wurde hinzugefügt und wurde durch ein Zahnrad auf der Startseite erreichbar.
+In diesen Einstellungen konnten die Nutzer nun den zu verwendenden Lautsprecher für die Wiedergabe auswählen und sich ausloggen.
+\paragraph*{Flow-Charts} $~$ \\
+\begin{wrapfigure}{r}{0cm}
+  \includegraphics[width=0.45\textwidth]{Settings.jpeg}
+  \caption{Leek Box Einstellungen}
+\end{wrapfigure}
+Auch war nun das Bearbeiten der Tags möglich.
```

Comment:
War nun möglich?
Wurde in diesem Milestone umgesetzt, oder?

Reply:
ja wurde